tests/points.test.js

```diff
@@ -0,0 +1 @@
+const points = require('../points.js');
```

Comment:
It is good practice to ensure that the last line of every file has an empty line. That is what that red circle is showing. Simply add a newline character at the end of line `1`(hit enter)